runtime/calamari/src/lib.rs

```diff
@@ -225,6 +225,9 @@ impl Contains<Call> for BaseFilter {
 			| Call::TechnicalMembership(_)
 			| Call::Scheduler(_)
 			| Call::CalamariVesting(_)
+			| Call::CollatorSelection(
+				manta_collator_selection::Call::register_as_candidate{..}
+				| manta_collator_selection::Call::leave_intent{..})
```

Comment:
@Dengjianping why only these 2 ?

Reply:
Both calls are open to all users. I think we should ban them for now.

Reply:
@Dengjianping I think like this you only allow those 2. Everything else is banned.
You need to list all the ones that are allowed here.